packages/modules/vehicles/polestar/auth.py

```diff
@@ -91,7 +90,7 @@ def _save_token_to_ramdisk(self) -> None:
             log.error("_save_token_to_ramdisk:error saving token store %s:%s", self.token_file, e)
 
     # auth step 3: get token
-    def get_auth_token(self) -> Optional[str]:
+    def get_auth_token(self) -> str or None:
```

Comment:
```suggestion
    def get_auth_token(self) -> Optional[str]:
```
Die Schreibweise folgt dem Python Style Guide.